koku/api/iam/serializers.py

```diff
@@ -18,22 +18,41 @@
 """Identity and Access Serializers."""
 # disabled module-wide due to meta-programming
 # pylint: disable=too-few-public-methods
+import random
+import string
+
 from django.core.validators import validate_email
 from django.db import transaction
 from rest_framework import serializers
 from rest_framework.validators import UniqueValidator
 
+from .email import new_user_reset_email
 from .models import Customer, ResetToken, User
 
 
+def generate_password():
```

Comment:
I think this is a temporary password (is it?) so it doesn't need-need to be as secure, but this is recommended for generating passwords https://docs.python.org/3.6/library/secrets.html instead of using random

Reply:
Yes, temporary. I'm gonna update the method name as well. But I'll switch over. Thanks for the info.